config/controlled_vocabulary.yml

```diff
@@ -1,5 +1,24 @@
 units:
   - Default Unit
+  - Aboriginal Multi-Media Society of Alberta (AMMSA)
+  - Anthropology, Department of
+  - Art/Research International
+  - Campus Data Summit Series
+  - Cultural and Pedagogical Inquiry
+  - Edmonton Mennonite Centre for Newcomers (EMCN)
+  - Graduate Studies and Research, Faculty of
+  - Grant Assist Program (Natural Sciences and Engineering)
+  - Grant Assist Program (Social Sciences / Humanities)
+  - Kule Folklore Centre
+  - Music, Department of
+  - Intonations
+  - Physics, Department of
+  - School of Library and Information Studies
+  - Secondary Education, Department of
+  - University of Alberta Archives
+  - University of Alberta Library
+  - University of Alberta Library Open Resources
```

Comment:
This one can be removed, my bad

Reply:
@seanluyk thanks, I've removed `Open Resources`